dbt_sugar/core/task/doc.py

```diff
@@ -158,9 +149,37 @@ def orchestrate_model_documentation(self, model_name: str, columns_sql: List[str
         not_documented_columns = self.get_not_documented_columns(content, model_name)
         self.document_columns(not_documented_columns)
 
-        self.update_column_descriptions(self.columns_to_update)
+        # Will remove from the update the test that doesn't run correctly.
+        self.check_tests(schema, model_name)
+        # Method to update the description test and tags from the model.
```

Comment:
not sure if we need that comment here I think the function name is pretty clear.